frontend/packages/dev-console/integration-tests/features/topology/quick_search_from_topology_views _to_add_to_project_ODC-2925

```diff
@@ -0,0 +1,73 @@
+Feature: Provide quick search from topology/list views to add to project 
+    As a user, when in Topology List/Graph view, I should be able to have a quick way to search for items to add to my app/project
+
+Background:
+    Given user is at developer perspective
+    And user is "aut-quick-search" namespace
+
+
+@regression
+Scenario: Quick search button in topology chart view
+    Given user is at topology view
+    When user clicks on chart view button
+    Then user can see quick search button
+
+
+@regression
+Scenario: Quick search button in topology list view
+    Given user is at topology view
+    When user clicks on list view button
+    Then user can see quick search button
+
+
+@regression
+Scenario: Quick search bar in topology chart view
+    Given user is at topology chart view
+    When user clicks Add to project button
+    Then user can see Add to project search bar
+
+
+@regression
+Scenario: Quick search bar in topology list view
+    Given user is at topology chart view
+    When user clicks Add to project button
+    Then user can see Add to project search bar
+
+
+@regression
+Scenario: Add django application in topology chart view
+    Given user is at topology chart view
+    When user clicks Add to project button
+    And user type django in Add to project search bar
+    And user select django+PostgreSQL option
+    And user clicks on instantiate template
+    And user clicks on create with default values in Instantiate Template form
+    And user can see postgres and django workload in topology chart view
+
+
+@regression
+Scenario: Add .Net application in topology list view
+    Given user is at topology list  view
+    When user clicks Add to project button
+    And user type "ab" in Add to project search bar
```

Comment:
```suggestion
    And user types "ab" in Add to project search bar
```

I would prefer enters instead of types. But upto you.

Reply:
Why will the user type ab to search for .Net? This scenario doesnt make sense. Change it to `user types net`.

Reply:
Updated